snap/hooks/configure

```diff
@@ -0,0 +1,84 @@
+#!/bin/sh -e
+
+# Confirm that the system is valid
+system="$(snapctl get system)"
+case "$system" in
+	udp4) ;;
+	udp6) ;;
+	tcp4) ;;
+	tcp6) ;;
+	*)
+		echo "'$system' is not a supported system" >&2
+		return 1
+		;;
+esac
+
+# Confirm that the middleware is valid
+middleware="$(snapctl get middleware)"
+case "$middleware" in
+        dds) ;;
+        rtps) ;;
+        ced) ;;
+        *)
+                echo "'$middleware' is not a supported middleware" >&2
+                return 1
+                ;;
+esac
+
+if [ "$system" = "serial" ] || [ "$system" = "pseudo-serial" ]; then
```

Comment:
```suggestion
if [ "$system" = "serial" ] || [ "$system" = "pseudoterminal" ]; then
```

Reply:
Ah, good catch, docs are a little out of date on this one.

Reply:
Fixed.

Reply:
Yes, sorry. That's on us. We are planning to update the documentation soon. Thanks for the highlight!